pkg/cfg/cfg.go

```diff
@@ -429,6 +452,11 @@ func setDefaultsGoogle() {
 	}
 }
 
+func setDefaultsADFS() {
+	log.Info("configuring ADFS OAuth")
+	OAuthopts = oauth2.SetAuthURLParam("resource", GenOAuth.RedirectURL) // Needed or all claims won't be included
+}
+
```

Comment:
I suspect this should be in #68 :)

Reply:
Yes, it should! I'm going to try and move it.

Reply:
@bnfinet I've been able to add it to #68, but I don't think I'll be able to create a PR for idToken without ADFS support (since that's what I'm using).